tests/Vagrantfile

```diff
@@ -0,0 +1,14 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "centos65"
+  config.vm.box_url = "http://downloads.juwai.io/devops/boxes/centos65-x86-64.box"
```

Comment:
At first we think with our own box we can put some commonly used stuff on it, and we can have a smaller size...  (for now I'm not sure if that's still the case.)